docs/x509.rst

```diff
@@ -652,6 +652,33 @@ X.509 Extensions
     purposes indicated in the key usage extension. The object is
     iterable to obtain the list of :ref:`extended key usage OIDs <eku_oids>`.
 
+.. class:: AuthorityKeyIdentifier
+
+    .. versionadded:: 0.9
+
+    The authority key identifier extension provides a means of identifying the
+    public key corresponding to the private key used to sign a certificate.
+
+    .. attribute:: key_identifier
+
+        :type: bytes
+
+        A value derived from the public key used to verify the certificate's
+        signature. See :rfc:`5280` section 4.2.1.2 for more details on the
+        recommended way to compute this value.
```

Comment:
Sorry, but I'm still not really liking this paragraph. It doesn't explain to me what I do if I just parsed a certificate with some value here